odoo/addons/test_search_panel_select_range/tests/test_search_panel_select_range.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+import odoo.tests
+
+
+@odoo.tests.tagged('post_install', '-at_install', 'dam')
```

Comment:
Selfish tag?  :wink:

Reply:
Will be removed of course. The branch is not yet finished.